app/src/pages/react-uswds-hello.tsx

```diff
@@ -0,0 +1,48 @@
+import {
+  Alert,
+  Button,
+  Grid,
+  GridContainer,
+  Icon,
+} from "@trussworks/react-uswds";
+import Link from "next/link";
+import React from "react";
+
+/** Example page showing use of react-uswds components. Can be deleted once you're on a roll. */
+const HelloWorld = () => {
+  return (
+    <GridContainer>
+      <Grid row gap={5}>
+        <Alert type="success" headingLevel="h1" heading="Hello world">
+          This is an alert
+        </Alert>
+      </Grid>
+      <Grid row>
+        <Grid col="auto">
+          <Icon.Build size={9} />
+        </Grid>
+        <Grid col="auto">
+          <p>
+            Here is some example text. You can learn more{" "}
+            <Link href="https://trussworks.github.io/react-uswds/">
```

Comment:
@sawyerh just to confirm, do you think it would be better to use the Link from react-uswds or from next/link? (Yesterday I just didn't find the react-uswds Link searching their storybook docs, but I see now I just missed it nested under Typography!). 

I also wasn't sure if we've aligned on a best practice since *I think*. most code bases I've worked in at Nava have used the next/link 🤔

Reply:
I think using the `next/link` and demonstrating adding USWDS classes to it could be helpful 👍🏻 That's likely what others familiar with Next.js will reach for, instead of react-uswds